x-pack/qa/sql/src/main/resources/string-functions.csv-spec

```diff
@@ -0,0 +1,30 @@
+bitLengthGroupByAndOrderBy
```

Comment:
Minor nitpick - put all functions into a `functions` file as oppose to `string-functions`.